index.html

```diff
@@ -11,6 +11,73 @@
 </head>
 <body>
 
+    <!--start nav section-->
+    <header>
+        <img class= "logo" src="img/karma-logo.svg"> 
+        <nav class="nav-section">
+        
+        <ul class="nav-section nav-section-second">
+            <li> <a class="nav-link active"href="#">Meet karma</a></li>
+            <li> <a class="nav-link active"href="#">How It Works</a></li>
+            <li> <a class="nav-link active"href="#">Store</a></li>
+            <li> <a class="nav-link active"href="#">Blog</a></li>
+            <li> <a class="nav-link active"href="#">Meet Help</a>"</li>
+            <li> <a class="nav-link active"href="#">Login</a></li>
+        </ul>
+        </nav>
+
+    </header>
+    <!--end nav section-->
+
+
+<!--main-->
+<main>
+<section class="hero">
+        <h1 class="hero-title"> Introducing Karma</h1>
+        <h2 class="hero-slogan">Bring WiFi with you, everywhere you go.</h2>
+        <a href="#"type="button" class="btn-hero">Leam More</a>
+    </section>
+<section class="section-info">
+    <h2 class="section-info-title">Everyone needs a little karma.</h2>
+    <div class="cards-container">
+        <div class="card">
+            <img src="img/icon-devices.svg"/>
+            <p> Internet for all devices</p>
+        </div>
+        <div class="card">
+            <img src="img/icon-coffee.svg"/>
+            <p> Boost your productivity</p>
+        </div>
+        <div class="card">
+            <img src="img/icon-refill.svg"/>
+            <p> Pay as you Go</p>
+        </div>
+
+    </div>
+</section>
+
+</main>
+<!--main-->
+
+<!--footer-->
+<footer class="footer">
+    <p class="footer-heading">Join us on</p>
+    <div class="box-social">
+        <div class="card-social">
+            <img src="img/twitter-icon.svg"/>
+             </div>
+
+             <div class="card-social">
+                <img src="img/facebook-icon.svg"/>
+                 </div>
+
+                 <div class="card-social">
+                    <img src="img/instagram-icon.svg"/>
+                     </div>
+    </div>
+
+    <p class="footer-copyright-text">&copy; karma Mobility,Inc.</p> 
+</footer>
     <!-- Add your HTML markup here -->
```

Comment:
Doing a brillant job with the semantic HTML throughout this page! nice that you've used some comment as well